cityofaustin/atd-moped
# Restores localstorage column config

## Associated issues
Restores the local storage of project list column visibility which we lost during the MUI 5 upgrade. 

## Testing
**URL to test:** Local or Netlify


**Steps to test:**
1. On the project list, use the column visibility toggle to hide some columns.
2. Navigate away from the project list, then return to it.
3. Observe that your column visibility settings are persisted.

---
#### Ship list
